protocol-impl/src/main/java/io/zeebe/protocol/impl/record/CopiedRecord.java

```diff
@@ -149,6 +150,6 @@ public String toJson() {
 
   @Override
   public String toString() {
-    return toJson();
+    return StringUtil.limitString(toJson(), 1024);
```

Comment:
What is this? maybe just serializing metadata?

Reply:
This is used by the `RecordingExporter`. If the value is too long, I don't want to spam the log or my IDE ;) But in general, the value is also interesting.

Reply:
hm ok